README.md

```diff
@@ -83,6 +78,8 @@ Create the application:
 ```
 $ cd kafka-streams-on-heroku
 $ heroku apps:create <application name>
+$ heroku buildpacks:add heroku/ruby
+$ heroku buildpacks:add heroku/java
```

Comment:
This needs to be `heroku/gradle` (https://github.com/heroku/heroku-buildpack-gradle).

Reply:
Updated.